opentelemetry-metrics-exporters/src/main/java/com/cumulocity/exporters/platform/PlatformMetricExporter.java

```diff
@@ -0,0 +1,107 @@
+package com.cumulocity.exporters.platform;
+
+import com.cumulocity.rest.representation.application.microservice.MicroserviceBillingRepresentation;
+import com.cumulocity.sdk.client.RestConnector;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.metrics.InstrumentType;
+import io.opentelemetry.sdk.metrics.data.AggregationTemporality;
+import io.opentelemetry.sdk.metrics.data.DoublePointData;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.metrics.export.MetricExporter;
+import lombok.extern.slf4j.Slf4j;
+import org.joda.time.DateTime;
+import org.springframework.context.ApplicationEventPublisher;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static io.opentelemetry.sdk.metrics.data.MetricDataType.*;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON_TYPE;
+
+
+@Slf4j
+public class PlatformMetricExporter implements MetricExporter {
+
+
+    private RestConnector restConnector;
+
+    private ApplicationEventPublisher applicationEventPublisher;
+
+    public PlatformMetricExporter(RestConnector restConnector, ApplicationEventPublisher applicationEventPublisher) {
```

Comment:
This class is not made a component as the TenantScope required for the RestConnector is lost when the Exporter is invoked by the running PeriodicMetricReader Scheduler. Hence this constructor is created, which can be used in the PlatformOpenTelemetryConfiguration